lib/studies/shrine/routes.dart

```diff
@@ -0,0 +1,2 @@
+const String loginRoute = '/shrine/login';
```

Comment:
Looks like this file is missing copyright header

Reply:
Done.